CMakeLists.txt

```diff
@@ -34,6 +34,12 @@ if (NOT CMAKE_BUILD_TYPE)
     "Choose the build type: Debug Release" FORCE)
 endif (NOT CMAKE_BUILD_TYPE)
 
+# debug flags
+if(CMAKE_BUILD_TYPE MATCHES Debug)
```

Comment:
Move this to the appropriate compiler function in this file: https://github.com/OpenFAST/openfast/blob/master/cmake/FastFortranOptions.cmake#L68